pages/blog/[article].js

```diff
@@ -46,7 +47,16 @@ function BlogArticle({ articleName }) {
     <>
       <Head title="Blog" />
       <HeroBanner title="Blog" className="smallHero" />
-      <Content theme="gray" columns={[<Article />]} />
+      <Content
+        theme="gray"
+        columns={[
+          <div>
+            <p className={styles.article}>
+              <Article />
+            </p>
+          </div>,
```

Comment:
Can you just do a `div` here and apply the styles to that?? Under the hood the Article is a markdown article where it might have elements that dont semantically belong as children of a `p` element.

Reply:
on it =)